ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -198,9 +200,44 @@ export const useNewMhrRegistration = () => {
   const fetchMhRegistrations = async (): Promise<void> => {
     const draftFilings = await getMhrDrafts()
     const myMhrHistory = await mhrRegistrationHistory()
-    setMhrTableHistory([...draftFilings, ...myMhrHistory])
+    const filteredMhrHistory = addHistoryDraftsToMhr(myMhrHistory, draftFilings)
+    setMhrTableHistory([...filteredMhrHistory])
   }
 
+  function addHistoryDraftsToMhr (mhrHistory: MhRegistrationSummaryIF[], mhrDrafts: MhrDraftTransferApiIF[]):
+    MhRegistrationSummaryIF[] {
+    const sorter = require('lodash')
+    const sortedDraftFilings = sorter.orderBy(mhrDrafts, ['createDateTime'], ['desc'])
+    // add drafts to Registrations..
+    const filteredMhrHistory = sorter
+      .orderBy(mhrHistory.filter(t => t.registrationDescription === 'REGISTER NEW UNIT'), ['createdDateTime'], ['desc'])
+    filteredMhrHistory.forEach(transfer => {
+      transfer.baseRegistrationNumber = transfer.mhrNumber
+      var mhrDrafts = sortedDraftFilings.filter(s => s.mhrNumber === transfer.mhrNumber)
+      if (mhrDrafts?.length > 0) {
+        transfer.hasDraft = true
+        transfer.changes = []
+        mhrDrafts.forEach(draft => {
+          const newDraft: MhRegistrationSummaryIF = {
+            mhrNumber: transfer.mhrNumber,
+            baseRegistrationNumber: transfer.mhrNumber,
+            submittingParty: draft.submittingParty,
+            clientReferenceId: transfer.clientReferenceId,
+            createDateTime: draft.createDateTime,
+            error: draft.error,
+            registrationDescription: draft.registrationDescription,
+            hasDraft: true,
+            ownerNames: '',
+            path: draft.path,
```

Comment:
Would it be easier here to spread the `draft` and include any additional information? Or is there specific information in the draft you don't want?
ie `...draft, newProperties: newValues` etc

Reply:
I guess its a healthy mix of both transfer/draft. 
No action required here

Reply:
Ok..